torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,41 @@
+import io
+
+import torch
+
+from typing import Union
+
+
+def dynamo_export(
+    torch_model: torch.nn.Module,
+    *args,
+    opset_version: Optional[int] = None,
+    options: Optional[ExportOptions] = None,
```

Comment:
![image](https://user-images.githubusercontent.com/5469809/225991591-e5211e62-7e44-43a4-937d-cd9b1fe1d932.png)

Reply:
### Name collision problem

#### Attempt 1: model input construct
```python
from typing import Tuple, Mapping
class ModelInput:
    args: Tuple
    kwargs: Mapping

    def __init__(self, *args, **kwargs):
        self.args = args
        self.kwargs = kwargs


dynamo_export(model, ModelInput(*args, **kwargs), ...).save("model.onnx")
```

#### Attempt 2: Return Exporter
```python
class Exporter:
    ...
    def export(self, *args, **kwargs) -> ExportOutput:
        ...

    def __call__(self, *args, **kwargs) -> ExportOutput:
        ...

def dynamo_exporter(model, options: ExportOptions) -> Exporter:
    ...

dynamo_exporter(model, options).export(*args, **kwargs).save("model.onnx")
dynamo_exporter(model, options)(*args, **kwargs).save("model.onnx")
```

Reply:
Name collision handling adds complexity to the API in the sense it becomes a two-step process (initialize and execute). Sacrificing the whole user experience (one step and done) for a corner case seems overkill. What do you think? @abock @BowenBao 

User can easily rename their `options` argument or wrap their model on an empty shell if they hit a collision.
Or we can use another name to make it unlikely to conflict e.g., `export_options`, `export_opts`